portfolio/src/main/java/com/google/sps/servlets/DeleteComment.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
```

Comment:
Please disregard this file. I was checking if I could retrieve the data in POST method. But there was no luck with this as we..

Reply:
Remove if possible.

Reply:
file has been removed. I forgot to stage the change last time.